functions/Copy-DbaDatabase.ps1

```diff
@@ -1214,7 +1214,7 @@ function Copy-DbaDatabase {
 								Write-Message -Level Verbose -Message "Successfully updated DatabaseOwnershipChaining for $sourceDbOwnerChaining on $dbName on $destination."
 							}
 							catch {
-								$copyDatabaseStatus.Status = "Successful - failed to apply DatabaseOwnershipChaining."
+								$copyDatabaseStatus.Status = "Successful - failed to apply DatabaseOwnershipChaining"
```

Comment:
The period needs to be put back, not sure why it was removed.

Reply:
I think other status messages lacked the period. I'll add back though. thanks